Source/diablo.cpp

```diff
@@ -616,6 +618,9 @@ static void diablo_init_screen()
 
 static void diablo_init()
 {
+	if (sgOptions.Graphics.bShowFPS)
+		EnableFrameCount();

```

Comment:
Why is the `-f` flag handling not in this same place? Shouldn't it be?

It seems bad that we have 2 flags for the same behavior, and they are enabled in different places.

Reply:
Flags are parsed before reading the settings.

Reply:
> Flags are parsed before reading the settings.

Still, can't we setup the FPS display in the same place? I assume we currently have 2 places in the code that call `EnableFrameCount`, which seems bad to me.

If we have the same setting exposed as a cmdline flag, and as an ini flag, those should be treated as different sources for the same setting and the enablement of it should be centralized.

Reply:
I think we should rethink this. Having the same behavior enabled in different places is a bug waiting to happen.